cosmo_tester/framework/test_hosts.py

```diff
@@ -636,58 +635,62 @@ def _update_instances_list(self, outputs):
-
-    def _clone_manager_blueprints(self):
-        self._manager_blueprints_path = git_helper.clone(
-                MANAGER_BLUEPRINTS_REPO_URL,
-                str(self._tmpdir))
-
-    def _create_inputs_file(self, manager):
-        inputs_file = self._tmpdir / 'inputs_{0}.json'.format(manager.index)
-        bootstrap_inputs = {
-            'public_ip': manager.ip_address,
-            'private_ip': manager.private_ip_address,
-            'ssh_user': self._attributes.centos_7_username,
-            'ssh_key_filename': self._ssh_key.private_key_path,
-            'admin_username': self._attributes.cloudify_username,
-            'admin_password': self._attributes.cloudify_password,
-            'manager_resources_package': self._manager_resources_package,
+            config_file = self._create_config_file(manager)
+            self._bootstrap_manager(manager, config_file)
+
+    def _create_config_file(self, manager):
+        config_file = self._tmpdir / 'config_{0}.yaml'.format(manager.index)
+        install_config = {
+            'manager':
+                {
+                    'public_ip': manager.ip_address,
+                    'private_ip': manager.private_ip_address,
+                    'security': {
+                        'admin_username': self._attributes.cloudify_username,
+                        'admin_password': self._attributes.cloudify_password,
+                    }
+                }
         }
 
         # Add any additional bootstrap inputs passed from the test
-        bootstrap_inputs.update(manager.bs_inputs)
-        bootstrap_inputs_str = json.dumps(bootstrap_inputs, indent=2)
-
-        self._logger.info(
-                'Bootstrap inputs:%s%s', os.linesep, bootstrap_inputs_str)
-        inputs_file.write_text(bootstrap_inputs_str)
-        return inputs_file
-
-    def _bootstrap_manager(self, inputs_file):
-        manager_blueprint_path = \
-            self._manager_blueprints_path / 'simple-manager-blueprint.yaml'
-        self._cfy.bootstrap([manager_blueprint_path, '-i', inputs_file])
+        install_config.update(manager.additional_install_config)
+        install_config_str = yaml.dump(install_config)
+
+        self._logger.info('Install config:\n{0}'.format(install_config_str))
+        config_file.write_text(install_config_str)
+        return config_file
+
+    def _bootstrap_manager(self, manager, inputs_file):
+        install_rpm_file = 'cloudify-manager-install.rpm'
+        with manager.ssh() as fabric_ssh:
+            fabric_ssh.run(
+                'curl -sS {0} -o {1}'.format(
+                    self._manager_install_rpm,
+                    install_rpm_file
+                )
+            )
+            fabric_ssh.sudo('yum install -y {0}'.format(install_rpm_file))
+            fabric_ssh.put(
+                inputs_file,
+                '/opt/cloudify-manager-install/config.yaml'
+            )
+            fabric_ssh.run('cfy_manager install')
```

Comment:
this doesn't need sudo, right?

Reply:
Yes. This depends on the user having sudo permissions (which we validate), but doesn't need to run as sudo.